test/test_framework.py

```diff
@@ -157,7 +157,9 @@ def on_foo(self, event):
         try:
             framework.observe(pub.baz, obs)
         except RuntimeError as e:
-            self.assertEqual(str(e), 'Observer method not provided explicitly and MyObserver type has no "on_baz" method')
+            self.assertEqual(str(e),
+                             'Observer method not provided explicitly'
+                             ' and MyObserver type has no "on_baz" method')
```

Comment:
I prefer the style you used below, not this "lines starting at random columns just because the opening "(" was there